.github/workflows/chromatic.yml

```diff
@@ -65,10 +67,11 @@ jobs:
       - name: Don't run job if user is not part of Isomer core team or trigger words are not present
         if: ${{ steps.checkUserMember.outputs.isTeamMember != 'true' || !(github.event_name == 'issue_comment' && github.event.issue.pull_request && steps.check.outputs.triggered == 'true') }}
         run: exit 0
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v3
+        with:
+          ref: refs/pull/${{ github.event.issue.number }}/head
```

Comment:
For this step, I think we can also do a `fetch-depth: 0` so that checking out will be faster, since we only need the files from the latest commit here.

Reply:
Hmm wait arent we already getting the `head` no? (effectively latest commit here?)

Reply:
Ah oops looks like I misunderstood the difference between `fetch-depth: 0` and `fetch-depth: 1`, the latter is what I was referring to (and looks like it is already done by default). Can ignore this!